pkgs/by-name/op/open-vm-tools/package.nix

```diff
@@ -129,6 +129,8 @@ stdenv.mkDerivation (finalAttrs: {
     "--with-fuse=fuse3"
   ] ++ lib.optional (!withX) "--without-x";
 
+  NIX_LDFLAGS = "-L${pam}/lib -lpam";
```

Comment:
Is this needed? pam is in the build inputs and configure should pick it up

Reply:
For me is better than symlink